app/src/main/java/com/moscow/tudee/presentation/screen/category/categoryTasksScreen/CategoryTasksScreen.kt

```diff
@@ -0,0 +1,245 @@
+package com.moscow.tudee.presentation.screen.category.categoryTasksScreen
+
+import android.net.Uri
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.moscow.tudee.R
+import com.moscow.tudee.domain.entity.Task
+import com.moscow.tudee.presentation.ObserveAsEvent
+import com.moscow.tudee.presentation.screen.category.CategoriesScreenState
+import com.moscow.tudee.presentation.screen.category.component.CategoryBottomSheet
+import com.moscow.tudee.presentation.screen.category.component.CategorySnackBar
+import com.moscow.tudee.presentation.screen.category.component.DeleteCategoryBottomSheet
+import com.moscow.tudee.presentation.screen.category.component.TabItem
+import com.moscow.tudee.presentation.screen.category.component.CategoryTabs
+import com.moscow.tudee.presentation.designSystem.component.*
+import com.moscow.tudee.presentation.designSystem.theme.Theme
+import com.moscow.tudee.presentation.screen.category.component.CategoryPriorityChip
+import org.koin.androidx.compose.koinViewModel
+import org.koin.core.parameter.parametersOf
+
+@Composable
+fun CategoryTasksScreen(
+    categoryId: Long,
+    navigateBack: () -> Unit,
+    navigateBackToCategoryScreen: (Int?) -> Unit
+
+) {
+    val viewModel: CategoryTasksViewModel = koinViewModel(parameters = { parametersOf(categoryId) })
+    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
+    ObserveAsEvent(viewModel.uiEvent) { event ->
```

Comment:
add line space between them
```suggestion
    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
    
    ObserveAsEvent(viewModel.uiEvent) { event ->
```